detekt-config.yml

```diff
@@ -0,0 +1,19 @@
+formatting:
+  active: true
+  NoWildcardImports:
+    active: false
```

Comment:
Vi burde kanskje standardisere på noe her så det er likt i alle appene?

Reply:
Ja, enten må vi finne på noe lurt for å kunne ha felles config, eller så bør vi bare kjøre default (uten) config

Fant ikke noen god måte å ha felles config på. Det gikk an å peke på en URL, men da blir vi avhengig av internett-forbindelse for å kjøre build. Er det kanskje OK, eller blir det uheldig?

Evnt noe for esyfo-cli, sette opp riktig detekt config på tvers?